src/fix/rgbfix.1

```diff
@@ -71,88 +80,84 @@ Fix the global checksum
 .It Cm G
 Trash the global checksum.
 .El
-.It Fl i Ar game_id
+.It Fl i , Fl Fl game-id Ar game_id
 Set the game ID string
 .Pq Ad 0x13F Ns \(en Ns Ad 0x142
 to a given string of exactly 4 characters.
-If both this and the title are set, the game ID will overwrite the
-overlapping portion of the title.
-.It Fl j
+If both this and the title are set, the game ID will overwrite the overlapping portion of the title.
+.It Fl j , Fl Fl non-japanese
 Set the non-Japanese region flag:
 .Ad 0x14A
 = 1.
-.It Fl k Ar licensee_str
+.It Fl k , Fl Fl new-licensee Ar licensee_str
 Set the new licensee string
 .Pq Ad 0x144 Ns \(en Ns Ad 0x145
 to a given string, truncated to at most two characters.
-.It Fl l Ar licensee_id
+.It Fl l , Fl Fl old-licensee Ar licensee_id
 Set the old licensee code,
 .Ad 0x14B ,
 to a given value from 0 to 0xFF.
 This value is deprecated and should be set to 0x33 in all new software.
-.It Fl m Ar mbc_type
+.It Fl m , Fl Fl mbc-type Ar mbc_type
 Set the MBC type,
 .Ad 0x147 ,
 to a given value from 0 to 0xFF.
-.It Fl n Ar rom_version
+.It Fl n , Fl Fl rom-version Ar rom_version
 Set the ROM version,
 .Ad 0x14C ,
 to a given value from 0 to 0xFF.
-.It Fl p Ar pad_value
+.It Fl p , Fl Fl pad-value Ar pad_value
 Pad the image to a valid size with a given pad value from 0 to 0xFF.
 .Nm
-will automatically pick a size from 32KiB, 64KiB, 128KiB, ..., 8192KiB and
-give a warning thereafter.
+will automatically pick a size from 32 kiB, 64 kiB, 128 kiB, ..., 8192 kiB.
```

Comment:
Needs to be KiB.

Reply:
I fixed this using a replace search, but the edit seems to have been lost somehow.